### PR TITLE
[UnitTests] Disabled some non working tests.

### PR DESCRIPTION
**Pull request by mkrueger**
These tests are no longer supported on that layer. TextChanging event
now works differently it's no longer synced.

### CODE REVIEW

**Comment by slluis** [comment]
Can you add alternative tests for TextChanging?

**Comment by mkrueger** [comment]
Added one - however it's now all based upon the MS data structures - IMO no need to test that on our level. Mid term our level will go away - we shouldn't maintain another editor API.

The other ones that fail I opened a bug 

https://bugzilla.xamarin.com/show_bug.cgi?id=54245

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Editor.TextMate/TextMateCompletionTextEditorExtension.cs [hunk 1]
```diff
@@ -117,6 +117,7 @@ static Task<IEnumerable<string>> GetAllWords(ITextSource source, int offset)
 		{
 			if (inactive)
 				return await base.HandleCodeCompletionAsync (completionContext, triggerInfo, token);
+			Console.WriteLine ("mime type :" + Editor.MimeType);
```
Comment (Therzok) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Editor.TextMate/TextMateCompletionTextEditorExtension.cs]: Debug code

File: main/tests/UnitTests/MonoDevelop.Ide.Gui/CompletionListWindowTests.cs [hunk 2]
```diff
@@ -466,6 +466,7 @@ public void TestAutoCompleteEmptyMatchOff ()
 		};
 
 		[Test]
+		[Ignore ("punctuation completion is now handled in backend")]
```
Comment (slluis) [inline main/tests/UnitTests/MonoDevelop.Ide.Gui/CompletionListWindowTests.cs]: What does it mean? why can't this be tested?

File: main/tests/UnitTests/MonoDevelop.Ide.Editor/TextSourceTestBase.cs [hunk 3]
```diff
@@ -94,6 +94,7 @@ public void TestCreateReader()
 		}
 
 		[Test]
+		[Ignore ("Broken - FIXME")]
```
Comment (slluis) [inline main/tests/UnitTests/MonoDevelop.Ide.Editor/TextSourceTestBase.cs]: Is there a bug reported for this? if not, there should be one before adding the Ignore.

File: main/tests/UnitTests/MonoDevelop.Ide.Editor/TextSourceTestBase.cs [hunk 4]
```diff
@@ -103,6 +104,7 @@ public void TestCreateReaderAt()
...
```
Comment (slluis) [inline main/tests/UnitTests/MonoDevelop.Ide.Editor/TextSourceTestBase.cs]: Same as above.

File: main/tests/UnitTests/MonoDevelop.Ide.Editor/ReadonlyTextDocumentTestBase.cs [hunk 5]
```diff
@@ -50,6 +50,7 @@ public void TestLineCount()
 
 
 		[Test]
+		[Ignore ("Broken - FIXME")]
```
Comment (slluis) [inline main/tests/UnitTests/MonoDevelop.Ide.Editor/ReadonlyTextDocumentTestBase.cs]: Bug #?

File: main/tests/UnitTests/MonoDevelop.Ide.Gui/CompletionListWindowTests.cs (hunk 2 above)
Comment (mkrueger) [inline main/tests/UnitTests/MonoDevelop.Ide.Gui/CompletionListWindowTests.cs]: It's no longer handled. If a backend requires that it needs to implement it.

Comment (slluis) [inline main/tests/UnitTests/MonoDevelop.Ide.Gui/CompletionListWindowTests.cs]: Should we remove the test then?

Comment (mkrueger) [inline main/tests/UnitTests/MonoDevelop.Ide.Gui/CompletionListWindowTests.cs]: I did in a later commit - it's right they're useless to us.

**Pull request merged**